cli/src/main/java/org/schlunzis/vigilia/cli/commands/VersionProvider.java

```diff
@@ -0,0 +1,13 @@
+package org.schlunzis.vigilia.cli.commands;
+
+import org.schlunzis.vigilia.cli.CLIApplication;
+import picocli.CommandLine;
+
+public class VersionProvider implements CommandLine.IVersionProvider {
+
+    @Override
+    public String[] getVersion() {
+        return new String[]{CLIApplication.class.getPackage().getImplementationVersion()};
```

Comment:
![image](https://github.com/user-attachments/assets/b9b83ea0-ba54-408e-a89a-1c0dbe4c263c)

Still not really working with the installed deb version... Don't know what's going on here...

Reply:
It's not even null, it's just empty...